src/entities/protocols/uniswap.ts

```diff
@@ -53,6 +68,11 @@ export class UniswapTrade implements Command {
   encode(planner: RoutePlanner, _config: TradeConfig): void {
     let payerIsUser = true
 
+    let safetyMode = this.options.safetyMode ?? SafetyModes.Normal
+    let inputETHSweep = safetyMode == SafetyModes.InputOutputSweeps || safetyMode == SafetyModes.AllSweeps
```

Comment:
maybe we rename to `shouldSweepETH` and `shouldSweepIntermediateTokens` for readability, know it's a bool flag